Tools/Scripts/webkitpy/style/main.py

```diff
@@ -154,7 +154,7 @@ def main(self):
             file_reader.do_association_check(host.scm().checkout_root)
         else:
             changed_files = paths if options.diff_files else None
-            patch = host.scm().create_patch(options.git_commit, changed_files=changed_files, git_index=options.git_index, find_branch=True)
+            patch = host.scm().create_patch(options.git_commit, changed_files=changed_files, git_index=options.git_index, commit_message=False, find_branch=True)
```

Comment:
It doesn't.

There are, AFAICT, two options here:

 * Make the style checker parse mboxes, and check the style for every commit message and every patch, or
 * Check the style of the total diff of all the commits.

That said, I wonder if we should make the webkitpy.common.checkout.scm.SCM.create_patch API less… odd. Because currently it returns either a unified diff or an mbox of commits, which the caller needs to parse in very different ways.

Reply:
filed https://bugs.webkit.org/show_bug.cgi?id=268403 for the API oddity